docs/source/blog/Improve-Audio-Transcriptions-with-Label-Studio.md

```diff
@@ -0,0 +1,287 @@
+              ]
+            }
+          }
+        ],
+        "score": 0.9722586794792298
+      }
+    ]
+  }
+]
+```
+The transcript has been shortened for readability. 
+
+## Set up Label Studio for audio transcription
+
+1. [Install Label Studio](/guide/install.html) using your desired method.
+2. Because the transcribed pre-annotations specify `s3://` URLs to the audio files in your S3 bucket, set an environment variable so that Label Studio creates presigned URLs to access the audio. Expose the following variable on the machine with Label Studio installed:
```

Comment:
add: It will use your aws credentials with s3 access policy configured before with `aws configure`